### PR TITLE
Refined HTMLOutputElement

### PR DESCRIPTION
**Pull request by phistuck**
This change is [Reviewable](https://reviewable.io/reviews/google/closure-compiler/1808)

### CODE REVIEW

File: externs/browser/html5.js [hunk 1]
```diff
@@ -3570,9 +3570,64 @@ HTMLDataListElement.prototype.options;
...
+/**
+ * @type {string}
+ */
+HTMLOutputElement.prototype.name;
+
+/**
+ * @const
+ * @type {!string}
+ */
+HTMLOutputElement.prototype.type = "output";
```
Comment (Dominator008) [inline externs/browser/html5.js]: Remove this assignment?

Comment (phistuck) [inline externs/browser/html5.js]: This was intentional and I saw similar stuff (a function with an actual function body that returns a string, for example). This is a constant string in a read only attribute. Why is this problematic?

Comment (ChadKillingsworth) [inline externs/browser/html5.js]: In general, externs should not have implementations. If they do - it's a mistake. It leads people to believe that the code actually executes.

Comment (phistuck) [inline externs/browser/html5.js]: Done.

**Pull request closed without merging**